util/test/testUtil.h

```diff
@@ -44,7 +45,7 @@ void valuecheck(const Eigen::DenseBase<DerivedA>& a, const Eigen::DenseBase<Deri
 
 void valuecheck(double a, double b, double tolerance = 1e-8)
 {
-  if (abs(a - b) > tolerance) {
+  if (std::abs(a - b) > tolerance) {
```

Comment:
i've updated my pull request that puts valuecheck into drakeUtil.  hopefully it's ready to go (just waiting unit tests).  
maybe you could manually merge in those changes into your branch, which will just force us to merge that pull request first.  let me know what you think.


On Apr 10, 2015, at 8:57 AM, Twan Koolen <notifications@github.com> wrote:

> In util/test/testUtil.h:
> 
> > @@ -44,7 +45,7 @@ void valuecheck(const Eigen::DenseBase<DerivedA>& a, const Eigen::DenseBase<Deri
> >  
> >  void valuecheck(double a, double b, double tolerance = 1e-8)
> >  {
> > -  if (abs(a - b) > tolerance) {
> > +  if (std::abs(a - b) > tolerance) {
> yeah, let's do the drakeMexUtil thing. Is it OK to keep valuecheck in testUtil? I also need to get rid of eigenToMatlab and matlabToEigen from testUtil; I'll do that right now.
> 
> —
> Reply to this email directly or view it on GitHub.
>